documentation/devonfw-ide-contribution-getting-started.asciidoc

```diff
@@ -5,12 +5,12 @@ toc::[]
 
 == Installation
 To start developing devonfw-ide, you must first install it on your computer.
-Therefore simply follow the link:setup.asciidoc[setup] guide.
+Therefore simply follow the link:setup.asciidoc[setup] guide. It is important to notice that this installation gives you the latest published version of the devon-ide with which you can execute commands like eclipse or docker whose scripts are located inside the "${DEVON_IDE_HOME}$/scripts/command, but for commiting code we use a differing folder structure. This will be addressed under xref:Contribution[Contribution].
 
 == Contribution
 To begin, read through the https://github.com/devonfw/.github/blob/master/CONTRIBUTING.asciidoc[Contribution Guidelines], which you should be sure to follow.
-First steps regarding forks or cloning of repos and creating branches, as well as some git commands, https://github.com/firstcontributions/first-contributions[can be found here].
-Clone the https://github.com/devonfw/ide[project IDE] at the beginning.
+First steps regarding forks or cloning of repos and creating branches, as well as some git commands, can be found inside https://github.com/firstcontributions/first-contributions[ this first contributions guideline].
+ To contribute code or text regarding the devon-ide at first you need to fork the https://github.com/devonfw/ide[project IDE] and can then clone your fork into your devon ide installations workspaces-folder (just make sure to execute 'git init' inside 'workspaces' before cloning). Here the https://github.com/firstcontributions/first-contributions[first contributions guideline] can be very helpful.
```

Comment:
Why should execute `git init`? That makes no sense to me.

Reply:
maybe write it like this: 
and then you can clone your fork into the folder `workspaces` in your devon-ide installation

Reply:
As it turned out, this was a kind of bug or eventually some config issue. In the folder I tested the clone process the clone command worked, but after executing `git init`. In other folders `git clone` seems to do just fine without `git init`.